lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -8487,20 +8497,37 @@ void EmitObjectInitializers(ParseNode *memberList, Js::RegSlot objectLocation, B
         while (memberList->nop == knopList)
         {
             ParseNode *memberNode = memberList->AsParseNodeBin()->pnode1;
-
-            if (memberNode->AsParseNodeBin()->pnode1->nop == knopComputedName)
-            {
-                useStore = true;
-            }
-
-            byteCodeGenerator->StartSubexpression(memberNode);
-            EmitMemberNode(memberNode, objectLocation, byteCodeGenerator, funcInfo, nullptr, useStore);
-            byteCodeGenerator->EndSubexpression(memberNode);
-            memberList = memberList->AsParseNodeBin()->pnode2;
+			if (memberNode->nop == knopEllipsis) 
+			{
+				//TODO[t-huyan]:emit spread
```

Comment:
Is this still a TODO, or does the `EmitObjectSpreadNode` call take care of that?